src/Features/LanguageServer/Protocol/Handler/IRequestHandlerMetadata.cs

```diff
@@ -14,6 +16,13 @@ internal interface IRequestHandlerMetadata
         /// <summary>
         /// Name of the language for LSP method to handle (optional).
         /// </summary>
-        string LanguageName { get; }
+        string? LanguageName { get; }
+
+        /// <summary>
+        /// Whether or not handling this method results in changes to the current solution state.
+        /// Mutating requests will block subsequent requests from starting until after they have
```

Comment:
can you clkarify if it blocks all subsequent requests, or just other mutating requests.